app/uk/gov/hmrc/gform/sharedmodel/formtemplate/Section.scala

```diff
@@ -56,6 +56,24 @@ object Section {
 
   implicit val format: OFormat[Section] =
     OFormatWithTemplateReadFallback(SectionTemplateReads.reads)
+
+  implicit val leafExprs: LeafExpr[Section] = new LeafExpr[Section] {
```

Comment:
Minor: can be converted to a lambda expression 

implicit val leafExprs: LeafExpr[Section] = (path: TemplatePath, t: Section) => t match { 
    case n: Section.NonRepeatingPage => LeafExpr(path, n.page)
}

Reply:
What would be the benefit of doing that?

Reply:
boilerplate

Reply:
I don't get this. Is that a joke?

Reply:
Nope, sorry if it came across so. Its a minor one, but looks compact

![image](https://user-images.githubusercontent.com/13782683/116369407-41f5e600-a801-11eb-8051-7173931a1a0d.png)

vs

![image](https://user-images.githubusercontent.com/13782683/116369493-59cd6a00-a801-11eb-8393-8dafe1231330.png)